spec/models/validation_report_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'spec_helper'
```

Comment:
Does the `ValidationReport` model need any tests on the `#find` and `#all` methods, or should this be removed?